libs/local_lcos/include/hpx/local_lcos/recursive_mutex.hpp

```diff
@@ -164,21 +164,21 @@ namespace hpx { namespace lcos { namespace local
                 return false;
             }
 
-//             bool try_timed_lock(threads::thread_id_type current_thread_id,
-//                 ::boost::system_time const& target)
-//             {
-//                 if (mtx.timed_lock(target))
-//                 {
-//                     locking_thread_id.exchange(current_thread_id);
-//                     recursion_count.store(1);
-//                     return true;
-//                 }
-//                 return false;
-//             }
+            //bool try_timed_lock(threads::thread_id_type current_thread_id,
+            //    ::boost::system_time const& target)
+            //{
+            //    if (mtx.timed_lock(target))
+            //    {
+            //        locking_thread_id.exchange(current_thread_id);
+            //        recursion_count.store(1);
+            //        return true;
+            //    }
+            //    return false;
+            //}
```

Comment:
@hkaiser same question with these: could we remove them? They seem to rely on boost functionality that we use nowhere else (and is perhaps even deprecated).

Reply:
I think those were meant to provide a compatibility layer with Boost timing facilities. We're past that stage, so let's remove them.